setup.py

```diff
@@ -52,11 +52,14 @@
       install_requires=[
           'setuptools',
           # -*- Extra requirements: -*-
+          'five.formlib',
       ],
       extras_require={
           'test': [
               'Products.CMFTestCase',
               'Products.PloneTestCase',
+              'plone.testing',
+              'plone.app.testing',
           ],
```

Comment:
They were needed because of https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/tests/PloneTestCase.py